pycromanager/acquisitions.py

```diff
@@ -394,6 +394,10 @@ def await_completion(self):
         if hasattr(self, '_event_thread'):
             self._event_thread.join()
 
+        # Close open datasets
+        if self._dataset is not None:
+            self._dataset.close()
+
```

Comment:
> I didn't see this--is it in another PR? Also, could you explain this a bit more? Did you do this generally or in the special case of the test?

This is the bit of code that I am referring to. This will close open datasets anytime `await_completion` is called. Are there cases where this behavior is not desired? The alternative would be to call `acq._dataset.close()` directly or to add a `acq.close_dataset()` method as a counterpart to `acq.get_dataset()`. What do you think?

Reply:
I think this could potentially create confusing behavior if you did something like this:

```python
with Acquistion(....
   dataset = acq.get_dataset()

# throws an error because dataset is closed?
dataset.read_image()
```

`acq.close_dataset()` might make sense as an addition. Or could you do this already with the existing API with `acq.get_dataset().close()`?

Reply:
Makes sense, I'll take this out then